codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/smithygo/awssdk/shapevisitor/ShapeVisitorHelper.java

```diff
@@ -0,0 +1,138 @@
+package software.amazon.polymorph.smithygo.awssdk.shapevisitor;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import software.amazon.polymorph.smithygo.codegen.GenerationContext;
+import software.amazon.polymorph.smithygo.codegen.GoWriter;
+import software.amazon.polymorph.smithygo.localservice.nameresolver.DafnyNameResolver;
+import software.amazon.polymorph.traits.ReferenceTrait;
+import software.amazon.smithy.model.shapes.MemberShape;
+import software.amazon.smithy.model.shapes.Shape;
+import software.amazon.smithy.model.traits.EnumTrait;
+
+public class ShapeVisitorHelper {
+
+  public static final Map<MemberShape, Boolean> toDafnyOptionalityMap =
```

Comment:
`SNAKE_CASE`

Reply:
This was updated to be a private variable. Not necessarily for private var